src/features/creators/newScriptProject.ts

```diff
@@ -0,0 +1,18 @@
+import { MarkdownString, window } from 'vscode';
+import { PythonProject, PythonProjectCreator, PythonProjectCreatorOptions } from '../../api';
+import { PythonProjectManager } from '../../internal.api';
+
+export class NewScriptProject implements PythonProjectCreator {
+    public readonly name = 'newScript';
+    public readonly displayName = 'Script';
+    public readonly description = 'Create a new Python script project';
```

Comment:
I wonder if the script scenario should just be "Python project"
```suggestion
    public readonly description = 'Create a new Python project';
```